packages/react/src/primitives/index.tsx

```diff
@@ -4,46 +4,6 @@ export * from './hooks';
 export * from './shared';
 export * from './types';
 
-export function Box(props) {
-  return <div data-amplify-box="" {...props} />;
-}
-
-export function Fieldset(props) {
-  return <fieldset data-amplify-fieldset="" {...props} />;
-}
-
-export function Footer(props) {
-  return <footer data-amplify-footer="" {...props} />;
-}
-
-export function Main(props) {
-  return <main data-amplify-main="" {...props} />;
-}
-
 export function Form(props) {
```

Comment:
Why do we still keep this?

Reply:
Because it's still being used in many of the Authenticator components. I'd prefer to remove it in a separate PR.

```bash
> src/components/Authenticator/SignUp/SignUp.tsx:4:23: error: No matching export in "src/index.tsx" for import "Form"
    4 │ import { Button, Flex, Form, View } from '../../..';
      ╵                        ~~~~
etc about 10 more times
```